src/mscorlib/src/System/BitConverter.cs

```diff
@@ -457,7 +457,31 @@ unsafe public static double ToDouble (byte[] value, int startIndex)
             // If we ever run on big endian machines, produce two versions where endianness is specified.
             Contract.Assert(IsLittleEndian, "This method is implemented assuming little endian with an ambiguous spec.");
             return *((double*)&value);
-        }                    
+        }
+
+        [SecuritySafeCritical]
+        public static unsafe int SingleToInt32Bits(float value)
+        {
+            // If we're on a big endian machine, what should this method do?  You could argue for
+            // either big endian or little endian, depending on whether you are writing to a file that
+            // should be used by other programs on that processor, or for compatibility across multiple
+            // formats.  Because this is ambiguous, we're excluding this from the Portable Library & Win8 Profile.
+            // If we ever run on big endian machines, produce two versions where endianness is specified.
+            Contract.Assert(IsLittleEndian, "This method is implemented assuming little endian with an ambiguous spec.");
+            return *((int*)&value);
+        }
+
+        [SecuritySafeCritical]
+        public static unsafe float Int32BitsToSingle(int value)
+        {
+            // If we're on a big endian machine, what should this method do?  You could argue for
```

Comment:
:question: Why include this comment? It's unlikely that the .NET Framework will ever support a system which uses different endianness for integers and floating point values (no one makes such a system anymore). In all other cases, this method behave as expected on both little- and big-endian machines.